esp/platform/espcontext.cpp

```diff
@@ -98,6 +101,8 @@ class CEspContext : public CInterface, implements IEspContext
     {
         m_txSummary.setown(new CTxSummary(m_creationTime));
         updateTraceSummaryHeader();
+        m_secureContext.setown(secureContext);
+        m_SecurityHandler.setSecureContext(secureContext);
```

Comment:
I know I expected lower case, but changing it seemed to be outside the scope of this issue. Otherwise, I would have to look at every file I touched to look for similar inconsistencies.